extensions/amp-story/1.0/amp-story-embedded-component.js

```diff
@@ -115,15 +115,16 @@ function getComponentSelectors(components, opt_predicate) {
 }
 
 /** @const {string} */
-const INTERACTIVE_ATTR_NAME = '[interactive]:not([interactive=false])';
+const INTERACTIVE_EMBED_SELECTOR = '[interactive=""]';
```

Comment:
I updated the selector here according to 

https://github.com/ampproject/amphtml/pull/21463#pullrequestreview-215877428

Reply:
Oh cool. Can you change it to simply `[interactive]` though?

Reply:
That would select attributes with values: https://jsfiddle.net/ryqokLn2/1/

Even if validation will not allow it I think we should be consistent and leave it like this.

Reply:
We don't need to check for things that aren't allowed by the validator though, but I don't feel strongly, so up to you : )